mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -2916,3 +2916,528 @@ def dataset_transform(self):
+            primaryClass={cs.CL}
+        },
+        """,
+        n_samples={"test": 367},
+        avg_character_length={"test": 551.07},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column("answer", "label")
+
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "text": example["Paragraph"]
+                + "\n\n"
+                + "Citation: "
+                + example["Citation"]
+            }
+        )
+
+
+class InsurancePolicyInterpretationLegalBenchClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="InsurancePolicyInterpretationLegalBenchClassification",
+        description="Given an insurance claim and policy, determine whether the claim is covered by the policy.",
+        reference="https://huggingface.co/datasets/nguha/legalbench",
+        dataset={
+            "path": "nguha/legalbench",
+            "name": "insurance_policy_interpretation",
+            "revision": "12ca3b695563788fead87a982ad1a068284413f4",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2000-01-01", "2023-08-23"),  # best guess
+        form=["written"],
+        domains=["Legal"],
+        task_subtypes=[],
+        license="cc-by-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        @misc{guha2023legalbench,
+            title={LegalBench: A Collaboratively Built Benchmark for Measuring Legal Reasoning in Large Language Models}, 
+            author={Neel Guha and Julian Nyarko and Daniel E. Ho and Christopher Ré and Adam Chilton and Aditya Narayana and Alex Chohlas-Wood and Austin Peters and Brandon Waldon and Daniel N. Rockmore and Diego Zambrano and Dmitry Talisman and Enam Hoque and Faiz Surani and Frank Fagan and Galit Sarfaty and Gregory M. Dickinson and Haggai Porat and Jason Hegland and Jessica Wu and Joe Nudell and Joel Niklaus and John Nay and Jonathan H. Choi and Kevin Tobia and Margaret Hagan and Megan Ma and Michael Livermore and Nikon Rasumov-Rahe and Nils Holzenberger and Noam Kolt and Peter Henderson and Sean Rehaag and Sharad Goel and Shang Gao and Spencer Williams and Sunny Gandhi and Tom Zur and Varun Iyer and Zehua Li},
+            year={2023},
+            eprint={2308.11462},
+            archivePrefix={arXiv},
+            primaryClass={cs.CL}
+        },
+        """,
+        n_samples={"test": 133},
+        avg_character_length={"test": 521.88},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column("answer", "label")
+
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "text": example["policy"] + "\n\n" + "Claim: " + example["claim"]
+            }
```

Comment:
Can we reformulate this as natural text?

Reply:
Should claim come first

Reply:
I think the order is okay, since the claim is based on the policy. I don't think formulating this as natural text is possible, since a clear distinction between the policy and claim needs to be made. The authors used the following prompt format:
```
Policy: {policy}
Claim: {claim}
```
Please see this for reference: [Insurance Policy Interpretation - base prompt](https://github.com/HazyResearch/legalbench/blob/main/tasks/insurance_policy_interpretation/base_prompt.txt).

Reply:
Right. Let us keep it as is.